swagger_to_sdk_config.json

```diff
@@ -1,16 +1,20 @@
 {
   "meta": {
     "after_scripts": [
-      "gofmt -w ./services/"
+      "gofmt -w ./services/",
+      "go get -u github.com/Azure/azure-sdk-for-go/tools/profileBuilder",
+      "profileBuilder -s list -l ./profiles/2017-03-09/defintion.txt -name 2017-03-09",
+      "profileBuilder -s preview -name preview",
+      "profileBuilder -s latest -name latest"
     ],
     "autorest_options": {
       "use": "@microsoft.azure/autorest.go@preview",
```

Comment:
Which version of the generator is this?

Reply:
This ties it to be the one most recently published from the [Azure/autorest.go](https://github.com/Azure/autorest.go) repository.

Reply:
v2 or v3?

Reply:
Good question, I believe we should tie it to v2. This falls outside of the scope of this PR though. :) For now I'm just trying to keep profiles up-to-date with the services folder.

Reply:
I agree it should be tied to the v2 generator but it's not clear to me if the moniker you're using will do that.

Reply:
I don't disagree, I'm happy to submit another PR to update that.

Reply:
OK looking at the following output I believe that preview is currently tied to the v2 branch so this should be safe.
```
npm view @microsoft.azure/autorest.go

{ name: '@microsoft.azure/autorest.go',
  description: 'The Go extension for classic generators in AutoRest.',
  'dist-tags': { latest: '2.0.24', preview: '2.1.69'
```